statwrap/fpp.py

```diff
@@ -539,3 +539,23 @@ def histogram(*data_args, class_intervals=None, bins=None, density=True, xlim=No
 
 def fpp_setup():
     apply_pd_changes()
+
+def contingency_table(data, column, row):
```

Comment:
@tzwickk are `column` and `row` reversed here? From your example, you have birth continent values as the different rows and it is in the `column` param place.